examples/demo.ipynb

```diff
@@ -106,7 +112,7 @@
     "from cleanvision.imagelab import Imagelab\n",
     "\n",
     "# Path to your dataset, you can specify your own dataset path\n",
-    "dataset_path = \"./image_files/\"\n",
+    "dataset_path = \"./image_files\"\n",
```

Comment:
just want to confirm removing the / at end was intentional

Reply:
No, I will revert it.